core/js/src/typespecs/app_types.ts

```diff
@@ -341,6 +352,28 @@ const patchDatasetSchema = createDatasetSchema
   .strict()
   .openapi("PatchDataset");
 
+const createPromptSchema = z
+  .object({
```

Comment:
nit: a shorter-hand way to write this would be `createPromptSchema = promptSchema.omit({ id: true })`